backend/src/services/pdf_generator.py

```
@@ -143,7 +146,9 @@ def _create_custom_styles(self):
             textColor=colors.HexColor('#666666')
         ))
     
-    async def generate(self, trip_id: str, itinerary: Dict, recommendations: Dict, enhanced_guide: Dict = None) -> str:
+    async def generate(self, trip_id: str, itinerary: Dict, 
+                      recommendations: Dict, 
+                      enhanced_guide: Dict = None) -> str:
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)